src/terra-core/TerraCoreDataModel.ttl

```diff
@@ -3112,11 +3058,11 @@ TerraCore:usesAntibody
   rdfs:domain TerraCore:AssayActivity ;
   rdfs:label "usesAntibody" ;
   rdfs:range [
-    owl:unionOf (
-      xsd:string
-      TerraCore:Antibody 
-    );
-  ];
+      owl:unionOf (
+          xsd:string
```

Comment:
can't unionOf object with datatype

Reply:
Deferred as there are many of these - See[ DM-507](https://broadworkbench.atlassian.net/browse/DM-507)